src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -123,23 +123,19 @@ std::string jsonRequestTreat
     break;
 
   case SubscriptionsRequest:
+    answer = parseSubscription(ciP, &parseDataP->subsV2);
+    if (answer != "OK")
     {
-
-      answer = parseSubscription(ciP, &parseDataP->subsV2);
-      if (answer != "OK")
-      {
-        return answer;
-      }
+      return answer;
     }
     break;
 
   case IndividualSubscriptionRequest:
+    answer = parseSubscription(ciP, &parseDataP->subsV2, true);  // NOTE: partial == true
+    LM_W(("httpInfo.extended: %s", parseDataP->subsV2.notification.httpInfo.extended? "true" : "false"));
+    if (answer != "OK")
     {
-      answer = parseSubscription(ciP, &parseDataP->subsV2, true);  // NOTE: partial == true
-      if (answer != "OK")
-      {
-        return answer;
-      }
+      return answer;
     }
     break;
 
```

Comment:
Discussed at skype. At least we have a plan to test next time this happens :)

NTC